src/harness/testcases/WINNF_FT_S_PPR_testcase.py

```diff
@@ -0,0 +1,391 @@
+    device_1['installationParam']['latitude'] = 38.8203
+    device_1['installationParam']['longitude'] = -97.2741
+
+    device_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    # Moving device_2 to a location outside 40 meters of PPA zone
+    device_2['installationParam']['latitude'] = 38.8201
+    device_2['installationParam']['longitude'] = -97.27651
+
+    device_3 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    # Moving device_3 to a location within PPA zone
+    device_3['installationParam']['latitude'] = 38.82767
+    device_3['installationParam']['longitude'] = -97.20497
+
+    device_4 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_d.json')))
+    # Moving device_4 to a location within PPA zone
+    device_4['installationParam']['latitude'] = 38.79583
+    device_4['installationParam']['longitude'] = -97.25303
+
+    # Load Grant requests
+    grant_request_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_request_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_1.json')))
+    grant_request_3 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_2.json')))
+    grant_request_4 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+
+    # device_b and device_d are Category B
+    # Load Conditional Data
+    self.assertEqual(device_2['cbsdCategory'], 'B')
+    conditionals_device_2 = {
+        'cbsdCategory': device_2['cbsdCategory'],
+        'fccId': device_2['fccId'],
+        'cbsdSerialNumber': device_2['cbsdSerialNumber'],
+        'airInterface': device_2['airInterface'],
+        'installationParam': device_2['installationParam'],
+        'measCapability': device_2['measCapability']
+    }
+
+    self.assertEqual(device_4['cbsdCategory'], 'B')
+    conditionals_device_4 = {
+        'cbsdCategory': device_4['cbsdCategory'],
+        'fccId': device_4['fccId'],
+        'cbsdSerialNumber': device_4['cbsdSerialNumber'],
+        'airInterface': device_4['airInterface'],
+        'installationParam': device_4['installationParam'],
+        'measCapability': device_4['measCapability']
+    }
+
+    conditionals = [conditionals_device_2, conditionals_device_4]
+
+    # Remove conditionals from registration
+    del device_2['cbsdCategory']
+    del device_2['airInterface']
+    del device_2['installationParam']
+    del device_2['measCapability']
+    del device_4['cbsdCategory']
+    del device_4['airInterface']
+    del device_4['installationParam']
+    del device_4['measCapability']
+
+    # Load PPA record
+    ppa_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'ppa_record_0.json')))
+    pal_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+
+    pal_low_frequency = 3550000000
+    pal_high_frequency = 3570000000
```

Comment:
pal_high_frequency = 3570000000. 

- Needs to be 3560000000 ? 
(Same in PPR.2)

Reply:
Modified   pal_high_frequency to 3570000000 for both testcases